app/components/edit/tab_form/pane_component.html.erb

```diff
@@ -5,11 +5,6 @@
       <% if help_text %><p><%= help_text %></p><% end %>
       <%= content %>
     </div>
-    <% if render_footer? %>
```

Comment:
The footer buttons are in the more specific components for collection and works.